### PR TITLE
add Ansible Playbook runner controls for Satellite

### PR DESCRIPTION
**Pull request by jharting**
Without runner:
screenshot_20180123_094509

With runner:
screenshot_20180123_094545
With runner (expanded):
screenshot_20180123_094605

### CODE REVIEW

File: app/js/portal/config.js [hunk 1]
```diff
@@ -32,6 +32,12 @@ function Config(cfpLoadingBarProvider, InsightsConfigProvider) {
     InsightsConfigProvider.setPortal(true);
     InsightsConfigProvider.setGettingStartedLink(
         'https://access.redhat.com/products/red-hat-insights#getstarted');
+
+    /* Used for Satellite integration
+    InsightsConfigProvider.setAnsibleRunner(function ($location, planId, customize) {
```
Comment (jharting) [inline app/js/portal/config.js]: @lphiri this is the hook for Satellite to provide which determines the URL of e.g. the job execution form

Comment (dLobatog) [reply app/js/portal/config.js]: Would this be a redirect or an API call?

Comment (jharting) [reply app/js/portal/config.js]: I am fine either way. Depends on what @lphiri prefers. Let me know if you need any other dependencies passed into the callback.

**Comment by iphands** [comment]
This resolves FMS work for #362

**Pull request merged**